package-lock.json

```diff
@@ -7548,7 +7548,7 @@
       "integrity": "sha1-jcrkcOHIirwtYA//Sndihtp15jc="
     },
     "repolinter": {
-      "version": "github:newrelic-forks/repolinter#226d44a82a81b37b45f781719ae6d25ce8d77a57",
+      "version": "github:newrelic-forks/repolinter#932b0553caa03ccd34a0481fe6243ec29c09f23f",
```

Comment:
Can we upgrade to the main todogroup version now that the PR is merged there? Or do you want to save that for a follow up?

Reply:
Let's save that for a follow up. I believe TODOGroup eventually would like to absorb this entire project, so my current plan is to switch to the TODOGroup version as part of the transition process. In the meantime it's super handy to be able to fix bugs without breaking anyone else's repolinter, so I'd like to keep the fork for now.

Reply:
As a reminder, I created an issue regarding this change: https://github.com/newrelic/repolinter-action/issues/5